docs/reference/api/docker_remote_api_v1.20.md

```diff
@@ -387,6 +389,7 @@ Return low-level information on the container `id`
 			"LxcConf": [],
 			"Memory": 0,
 			"MemorySwap": 0,
+			"KernelMemory": 0,
```

Comment:
Changes in this document have to be moved to `docker_remote_api_v1.21.md` now (for docker 1.9)